src/styles/components/_component-grid.scss

```diff
@@ -1,13 +1,26 @@
 $breakpoint-tablet: 820px;
 
-.components-grid {
+.grid-container {
   width: 100%;
-  text-align: center;
-  padding: 1rem;
 
   @include background-pattern;
+}
+
+.components-grid {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  margin: auto;
```

Comment:
If this is just for horizontally centering the grid, `margin-inline: auto` is usually a better way to go, since it leaves the block (vertical) direction alone.